build_tools/aws-sdk-code-generator/spec/protocol-tests-ignore-list.json

```diff
@@ -0,0 +1,98 @@
+{
+  "ec2" :
+    {
+      "input" : [
+        "Ec2QuerySupportsNaNFloatInputs",
```

Comment:
Will we be working through this list or is it set in stone?

Reply:
Half n half. I plan to do an initial run of fixes and trace back to do some in-depth cases (like the `NaN`, `Infinity`, etc) in a separate PR.